pkg/resources/pipe_grant.go

```diff
@@ -89,26 +89,33 @@ func PipeGrant() *TerraformGrantResource {
 
 // CreatePipeGrant implements schema.CreateFunc.
 func CreatePipeGrant(d *schema.ResourceData, meta interface{}) error {
-	var pipeName string
+	var (
+		pipeName   string
+		schemaName string
+	)
 	if name, ok := d.GetOk("pipe_name"); ok {
 		pipeName = name.(string)
 	}
+	if name, ok := d.GetOk("schema_name"); ok {
+		schemaName = name.(string)
+	} else {
+		schemaName = ""
```

Comment:
FYI you don't need to set schemaName to an empty string. Since you already declared it on line 94, and the default value of a type string is an empty string.